server/lib/tuist/runs.ex

```diff
@@ -139,6 +139,38 @@ defmodule Tuist.Runs do
     |> Flop.validate_and_run!(attrs, for: Build)
   end
 
+  def recent_build_status_counts(project_id, limit \\ 40) do
```

Comment:
should the `asc` vs `desc` be passed as an option instead of having separate methods?